docker-for-mac/install.md

```diff
@@ -91,21 +91,10 @@ and select **Learn**.
 
 ## Automatic updates
 
-Starting with Docker Desktop 3.0.0, updates to Docker Desktop will be available automatically as delta updates from the previous version.
-
-When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can start downloading the update in the background whenever it is convenient for you.
-
-After downloading the update, all you need to do is to click **Update and restart** from the Docker menu. This installs the latest update and restarts Docker Desktop for the changes to take effect.
-
-## Uninstall Docker Desktop
-
-To uninstall Docker Desktop from your Mac:
-
-1. From the Docker menu, select **Troubleshoot** and then select **Uninstall**.
-2. Click **Uninstall** to confirm your selection.
```

Comment:
I don’t think this was meant to be removed, was it?

Reply:
Good catch. Fixed it 😅